index.html

```diff
@@ -0,0 +1,264 @@
+                                    fill="#505F79" />
+                            </svg>
+                        </a></li>
+                </ul>
+                <button class="about-btn" type="submit">Get my resume</button>
+            </div>
+            <!-- My Skills -->
+            <div class="skills">
+                <ul>
+                    <li class="p">Languages
+                        <img src="img/Union.svg" alt="counter" class="counter">
+                        <ul class="lang">
+                            <li class="skill"><img src="img/javascript.svg" alt="">JavaScript </li>
+                            <li class="skill"><img src="img/html.svg" alt="">HTML</li>
+                            <li class="skill"><img src="img/css.svg" alt="">CSS</li>
+                        </ul>
+                    </li>
+                    <li class="p">Frameworks <img src="img/Union-right.svg" alt="counter" class="counter"></li>
+                    <li class="p hr">Skills <img src="img/Union-right.svg" alt="counter" class="counter"></li>
```

Comment:
- [ ] As per the previous reviewer, :thinking: I think it would be better to stick to the figma design, giving the proper styles to this section, I suggest to set the list-style to none and apply the margins and padding on the figma design.

 - Your design:
![stacksY](https://user-images.githubusercontent.com/97834160/206459356-a62efc5b-0956-4f6b-9d7c-d0437ee7e0f1.png)
 - Figma design:
![stacksF](https://user-images.githubusercontent.com/97834160/206459440-57036103-3d63-49b9-a15c-32a87e024c99.png)